Bookish/Bookish.Web/Views/Home/Catalogue.cshtml

```diff
@@ -24,7 +24,7 @@
                 <td>@book.authors</td>
                 <td>@book.isbn</td>
                 <td>
-                    @Html.ActionLink("Borrow","Book", "Home")
+                    <a type="button" class="btn btn-info" asp-controller="Home" asp-action="Book">Borrow</a>
```

Comment:
I don't think you need `asp-controller`? The home controller should be the default 🙂